docker/planetscale_proxy/Dockerfile

```diff
@@ -1,8 +1,8 @@
 FROM golang:1
 
 RUN apt update && apt install netcat-openbsd -y
-RUN cd /go/src && git clone https://github.com/mattrobenolt/ps-http-sim.git
-RUN cd /go/src/ps-http-sim && go install .
+RUN cd /go/src && git clone https://github.com/prisma/planetscale-proxy.git
+RUN cd /go/src/planetscale-proxy && go install .
 
 ENTRYPOINT /go/bin/ps-http-sim \
```

Comment:
might be worth a comment that the name comes from the codebase above (which now has no more connection in the name itself)